prometheus_client/openmetrics/exposition.py

```diff
@@ -28,8 +36,8 @@ def generate_latest(registry):
                 else:
                     labelstr = ''
                 if s.exemplar:
-                    if metric.type not in ('histogram', 'gaugehistogram') or not s.name.endswith('_bucket'):
-                        raise ValueError("Metric {0} has exemplars, but is not a histogram bucket".format(metric.name))
+                    if not _is_valid_exemplar_metric(metric, s):
+                        raise ValueError("Metric {0} has exemplars, but is not a counter or histogram bucket".format(metric.name))
```

Comment:
I'd say histogram bucket or counter

Otherwise it could be misread that counter buckets are a thing.